specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/ContainerApps.json

```diff
@@ -728,6 +731,25 @@
               "$ref": "./CommonDefinitions.json#/definitions/WorkloadProfileName",
               "description": "Workload profile name to pin for container app execution."
             },
+            "patchingConfiguration": {
+              "description": "Container App auto patch configuration.",
+              "type": "object",
+              "properties": {
+                "patchingMode": {
+                  "description": "Patching mode for the container app.",
+                  "type": "string",
+                  "enum": [
+                    "Automatic",
+                    "Manual",
+                    "Disabled"
+                  ],
+                  "x-ms-enum": {
+                    "name": "patchingMode",
+                    "modelAsString": true
```

Comment:
>modelAsString [](http://example.com/codeflow?start=21&length=13)

Consider adding more detailed documentation here to explain the difference between the modes.